src/containerapp/setup.py

```diff
@@ -39,7 +39,7 @@
 # TODO: Add any additional SDK dependencies here
 DEPENDENCIES = [
     'azure-cli-core',
-    'azure-mgmt-kubernetesconfiguration==2.0.0',
+    'azure-mgmt-kubernetesconfiguration==3.0.0b1',
```

Comment:
why do you need to use beta version sdk?

Reply:
revert to 2.0.0